components/support/ktx/src/test/java/mozilla/components/support/ktx/android/content/ContextTest.kt

```diff
@@ -44,4 +65,49 @@ class ContextTest {
 
         assertEquals(extensionFunctionResult, normalMethodResult)
     }
+
+    @Test
+    fun `isScreenReaderEnabled() returns true when TouchExploration enabled`() {
+        shadowOf(accessibilityManager).isTouchExplorationEnabled = true
+        context!!.isScreenReaderEnabled()?.let { assertTrue(it) }
```

Comment:
Tests should always make explicit checks. For this to pass `isScreenReaderEnabled` can return `null` or `true`. But the test pretends to check if it returns `true`.

Reply:
You are right.